pkg/products/rhssouser/reconciler.go

```diff
@@ -301,6 +303,12 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, installation *inte
 			Requests: corev1.ResourceList{corev1.ResourceCPU: k8sresource.MustParse("750m"), corev1.ResourceMemory: k8sresource.MustParse("1500Mi")},
 			Limits:   corev1.ResourceList{corev1.ResourceCPU: k8sresource.MustParse("1500m"), corev1.ResourceMemory: k8sresource.MustParse("1500Mi")},
 		}
+
+		//Set keycloak Update Strategy to Rolling as default
+		//Keycloak operator should make decision based on the image, and can change update strategy
+		//kc.Spec.Migration.MigrationStrategy = "rolling"
+		kc.Spec.Migration.MigrationStrategy = keycloak.MigrationStrategy(strings.ToLower(string(appsv1.DeploymentStrategyTypeRolling)))
```

Comment:
shouldn't the strategy be changed prior reconciling the subscription https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/rhssouser/reconciler.go#L187? 

cc: @matskiv

Reply:
I think this time around we don't need to do that because the SSO operator won't be updated.
In the future, we should change it before reconciling subscription if we upgrade to next minor version and need the "recreate" strategy. Unless keycloak-operator will take care of that for us.